data/sql/updates/pending_db_world/rev_1552859629415687000.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1552859629415687000');
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (31691, 33470, 0, 100, 0, 1, 0, 1, 7, NULL);
```

Comment:
Always `DELETE` before insert in order to make your query re-executable :)

In your case, something like

```sql
DELETE FROM `creature_loot_template` WHERE `Entry` = 31691 AND `Item` = 33470;
```